.github/workflows/ci.yml

```
@@ -262,22 +262,33 @@ jobs:
           base-ref: ${{ github.event.pull_request.base.sha || 'main' }}
           head-ref: ${{ github.event.pull_request.head.sha || github.ref }}
 
-  run-test-lab:
+  e2e:
     runs-on: ubuntu-22.04
     needs: [ build-ubuntu-X64 ]
     strategy:
       fail-fast: false
       matrix:
+        mode: ["std"]
         era: ${{ fromJSON(needs.build-ubuntu-X64.outputs.eras) }}
-        run_id: [1,2,3]
-        # Deactivate Conway era which is not currently working with the Pallas Chain Observer yet, value should be 5-10
-        hard_fork_latest_era_at_epoch: [1000] 
+        cardano_node_version: ["8.7.3"]
+        hard_fork_latest_era_at_epoch: [0,10,100]
+        run_id: ["#1","#2","#3"]
         extra_args: [""]
 
         include:
-          - era: ${{ fromJSON(needs.build-ubuntu-X64.outputs.eras)[0] }}
-            run_id: 3
-            hard_fork_latest_era_at_epoch: 1000
+          # Include a hard fork test for `8.1.2` which is not concerned`
```

Just the trailing character `
```suggestion
          # Include a hard fork test for `8.1.2` which is not concerned
```

---

Done :heavy_check_mark: